review-tools/ghmerge

```diff
@@ -168,30 +176,22 @@ if [ -z "$WHO" -o -z "$BRANCH" -o -z "$REPO" ]; then
     exit 1
 fi
 
-REPO_CHECK=$(python3 -c '
-from __future__ import print_function
-import json, sys
-rtm_set=0
-urgent_set=0
-input = json.load(sys.stdin)
-# Dont do this check if its not for the openssl repo
-if (input["base"]["repo"]["name"] != "openssl"):
-    sys.exit(0)
-for l in input["labels"]:
-    if (l["name"] == "approval: ready to merge"):
-        rtm_set=1
-    if (l["name"] == "severity: urgent"):
-        urgent_set=1
-if (rtm_set == 0 and urgent_set == 0):
-    print("Not ready to merge")
-' <$PR_URL_CONTENTS)
-
-if [ "$REPO_CHECK" == "Not ready to merge" ]
+TARGET_REPO=$(jq -r '.base.repo.name' $PR_URL_CONTENTS)
+RTM_LABEL=$(jq -r '.labels[] | select(.name == "approval: ready to merge") | .name' $PR_URL_CONTENTS)
+URGENT_LABEL=$(jq -r '.labels[] | select(.name == "severity: urgent") | .name' $PR_URL_CONTENTS)
+
+if [ "$TARGET_REPO" == "openssl" ]
```

Comment:
I would invert the condition here. then code can be changed like that
```
if [ "$TARGET_REPO" != "openssl" ] ;
then
   >&2 echo "Skipping ready to merge check for non-openssl repo"
elif [ -z "$RTM_LABEL" -a -z "$URGENT_LABEL" ] ;
then
  >&2 echo "This PR has neither the ready to merge or urgent label set, can't merge"
  exit 1
fi
```

Reply:
done